packages/account/src/Sections/Security/ConnectedApps/__tests__/connected-apps.spec.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { render, screen, waitFor, waitForElementToBeRemoved, fireEvent } from '@testing-library/react';
+import ConnectedApps from '../connected-apps';
+
+const oauth_apps_list = [
+    {
+        name: 'Local',
+        app_markup_percentage: 0,
+        app_id: 9999,
+        scopes: ['read', 'admin', 'trade', 'payments'],
+        last_used: '2021-10-31 06:49:52.054529',
+    },
+];
+
+jest.mock('@deriv/shared/src/services/ws-methods', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    WS: {
+        authorized: {
+            send: ({ oauth_apps, revoke_oauth_app }) => {
+                if (revoke_oauth_app) oauth_apps_list.pop();
+                return { oauth_apps: oauth_apps_list };
+            },
+        },
+    },
+}));
+
+const modalRoot = document.createElement('div');
+modalRoot.setAttribute('id', 'modal_root');
+document.body.appendChild(modalRoot);
+
+describe('Connected Apps', () => {
+    const originalOffsetHeight = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetHeight');
+    const originalOffsetWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetWidth');
+
+    beforeAll(() => {
+        Object.defineProperty(HTMLElement.prototype, 'offsetHeight', { configurable: true, value: 50 });
+        Object.defineProperty(HTMLElement.prototype, 'offsetWidth', { configurable: true, value: 50 });
+    });
+
+    afterAll(() => {
+        Object.defineProperty(HTMLElement.prototype, 'offsetHeight', originalOffsetHeight);
+        Object.defineProperty(HTMLElement.prototype, 'offsetWidth', originalOffsetWidth);
+    });
+
+    test('renders correctly', () => {
+        render(<ConnectedApps />);
```

Comment:
This requires assertion, how we can make sure that it renders correctly? 

Perhaps, by merging the test cases below, `shows the title correctly` and `loads and renders the list`.